src/main/java/journeymap/client/data/WorldData.java

```diff
@@ -46,6 +47,7 @@ public class WorldData extends CacheLoader<Class, WorldData>
 {
     String name;
     int dimension;
+    static private TIntObjectHashMap<String> dimNames;
```

Comment:
this should be `private static TIntObjectMap<String> dimNames;` use the interface as the variable not the implementation and private before static.